src/types/redis_json.cc

```diff
@@ -548,6 +548,66 @@ std::vector<rocksdb::Status> Json::MGet(const std::vector<std::string> &user_key
   return statuses;
 }
 
+rocksdb::Status Json::MSet(const std::vector<std::string> &user_keys, const std::vector<std::string> &paths,
+                           const std::vector<std::string> &values) {
+  std::optional<MultiLockGuard> guard;
```

Comment:
why it's an optional guard?

Reply:
Yes, this is an unnecessary expression, thank you very much for your suggestion, I will remove the optional.

I originally used optional here just because I noticed that it was used in the string.mset implementation, but in fact it was because of the possibility of not locking in string.mset. So there is no need to use optional in json.mset.